images/linux/scripts/installers/test-toolcache.sh

```diff
@@ -9,6 +9,22 @@
 # Fail out if any tests fail
 set -e
 
+# define array of key aliases
+key_alias_array=()
+bash --version
+# define dicionary for key_alias and its values
+declare -A toolcache_key_value
+PACKAGE_LIST=($(jq -r 'keys | .[]' "$INSTALLER_SCRIPT_FOLDER/toolcache.json"))
```

Comment:
we have a lot of `$(jq -r 'keys | .[]' "$INSTALLER_SCRIPT_FOLDER/toolcache.json")` occurrences in this script, maybe it makes sense to create a function for that?

Reply:
Test logic for Python, Ruby and PyPy was unified and put into single function.